resources/js/App/Components/LayoutLeft/KeywordList/KeywordList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./keywordlist.css";
+
+import Keyword from "./Keyword/Keyword.jsx";
+
+export default class KeywordList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      keywordList: [],
+    }
+  }
+
+  componentDidMount() {
+    fetch("api/keywords")
+      .then( (response) => { return (response.json()) } )
+      .then( (data) => { this.setState( { keywordList: data } ) } )
+  }
+
+  render() {
+    const { keywordList } = this.state;
+    return(
+      <div className="keywordlist">
+        <p>Writing from KeywordList</p>
+        {
+          keywordList.length > 0 
+          &&
+          <Keyword keywordList={keywordList} isVideoPageOn={this.props.isVideoPageOn}/>
```

Comment:
You have somewhat misleading naming for `KeywordList` and `Keyword`. Just from the name of those components, I would imagine that `Keyword` is rendering a single keyword, while `KeywordList` would loop through all the keywords. But you have a the looping part in the `Keyword`. Should change naming to be clearer. Or update the component to fit the naming.

Reply:
By the way, `isVideoPageOn` is also a weird name for a callback function. Usually, when property that starts with `is...` is a boolean. Like, "this property defined if my video page is ON". But in your case it's a callback function. A better name would be something like `onVideoPageChange`.